src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -31,10 +28,6 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
   const helperDetermineCommenterName = () => {
     const anonymousCommenterName = firstComment?.reaction?.data?.anon_user_info_emoji_name;
 
-    const firstCommenterId = firstComment?.reaction?.user_id;
-    const isFirstCommenterMe =
-      firstCommenterId === signedProfileId || firstCommenterId === anonProfileId;
-
     if (firstComment?.reaction?.isOwningReaction) {
       commenterName = 'You';
       postNotificationPicture = firstComment?.reaction?.user?.data?.profile_pic_url;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code seems to have removed the logic for checking if the first commenter is the current user. This might affect the functionality of the application if the `isOwningReaction` property does not accurately reflect whether the current user made the comment. Please ensure that this change does not introduce any bugs or unexpected behavior.
```diff
-    const firstCommenterId = firstComment?.reaction?.user_id;
-    const isFirstCommenterMe =
-      firstCommenterId === signedProfileId || firstCommenterId === anonProfileId;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->